### PR TITLE
Python Package

### PR DESCRIPTION
**Pull request by sdreyer**
Python Package:
- Currently missing readme
- can be run directly as featureform or as python3 -m featureform

### CODE REVIEW

File: ff/serving/gen_grpc.sh [hunk 1]
```diff
@@ -6,4 +6,6 @@ protoc --go_out=. --go_opt=paths=source_relative     --go-grpc_out=. --go-grpc_o
 python -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. ./proto/serving.proto
 
 protoc --go_out=. --go_opt=paths=source_relative     --go-grpc_out=. --go-grpc_opt=paths=source_relative     ./metadata/proto/metadata.proto
-python -m grpc_tools.protoc -I./metadata/proto --python_out=client --grpc_python_out=client metadata.proto
+python -m grpc_tools.protoc -I./metadata/proto --python_out=client/src/featureform --grpc_python_out=client/src/featureform metadata.proto
+pip install 2to3
+2to3 ./client/src/featureform/metadata_pb2_grpc.py -w
```
Comment (simba-git) [inline ff/serving/gen_grpc.sh:11]: Why?

File: ff/serving/client/src/featureform/cli.py [hunk 2]
```diff
@@ -88,6 +86,8 @@ def apply(host, insecure, files):
     print(register.state().sorted_list())
     register.state().create_all(stub)
 
-
+print("above cli")
```
Comment (simba-git) [inline ff/serving/client/src/featureform/cli.py:89]: Remove prints

File: ff/serving/client/setup.cfg [hunk 3]
```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = featureform
+version = 0.0.1
+author = Featureform Inc
+#author_email = author@example.com
```
Comment (simba-git) [inline ff/serving/client/setup.cfg:5]: hello@featureform.com

File: ff/serving/client/setup.cfg [hunk 4]
```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = featureform
+version = 0.0.1
```
Comment (simba-git) [inline ff/serving/client/setup.cfg:3]: This won't work

File: ff/serving/client/setup.cfg [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+author = Featureform Inc
```
Comment (simba-git) [inline ff/serving/client/setup.cfg:4]: FeatureForm, Inc.

File: ff/serving/client/setup.cfg [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+#author_email = author@example.com
+description = CLI Package for the Featureform Feature Store
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://featureform.com
+project_urls =
+    Bug Tracker = https://github.com/featureform/embeddinghub/issues
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: MIT License
```
Comment (simba-git) [inline ff/serving/client/setup.cfg:14]: MPL license

Comment (simba-git) [reply ff/serving/client/setup.cfg:14]: Also fix the license above

**Review by simba-git — CHANGES_REQUESTED** [review]
Also verify that if you push this to pypi then pull and try it it that it works as expected

File: ff/serving/gen_grpc.sh (hunk 1 above)
Comment (sdreyer) [reply ff/serving/gen_grpc.sh:11]: Workaround for the grpc pathing. Finally figured it out

File: ff/serving/client/setup.cfg [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+author_email = hello@featureform.com
+description = Package for the Featureform Feature Store
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://featureform.com
+project_urls =
+    Bug Tracker = https://github.com/featureform/embeddinghub/issues
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)4
```
Comment (simba-git) [inline ff/serving/client/setup.cfg:14]: Is that 4 at the end a typo

**Review by simba-git — APPROVED** [review]

**Pull request merged**